### ISSUE TITLE
Featured Researchers - Make it more snazzier

### ISSUE THREAD

**Issue by vantuyls** [projecthydra-labs/hyrax#1003]
### Descriptive summary

Featured Researcher area of the splash page of Hyrax is pretty static and a bore. We'd like to see a more dynamic and modifiable Featured Researcher area including:
- [ ] Allow admins to delete old featured researchers from the list
- [ ] Allow for the featured researcher on the homepage to rotate among a set of featured researchers
- [ ] Consider what other types of "marketing things" should/could be included in this area of the splash page (???)

@mjgiarlo

**Comment by mjgiarlo** [comment]
@vantuyls Tweaked the above. Pls review when you can. Thanks!

**Comment by vantuyls** [comment]
i think allowing the label to be whatever one wants is useful. In the context of our repository, we may want to feature a researcher OR a collection, depending on our state of mind at the time.

i also would like to add that the "see previous featured researchers" area needs to have the ability to delete previous entries so that it doesn't become an endless graveyard of old entries.

**Comment by hannahfrost** [comment]
+1 to all of these ideas. 

Tagging @ggeisler so he can factor these potential changes to the new home page designs he will be starting on soon.

**Comment by geekscruff** [comment]
I'd like to suggest allowing featured researchers to be a flip-flop in the dashboard - some people won't want this (some repos won't be for 'researchers' work).

**Comment by vantuyls** [comment]
@geekscruff good point. i think i'd made a similar one to @mjgiarlo a few weeks ago. Do you think your non-research repository would want "featured collections" or some other splash page marketing/promotion space? Would it be possible to make the "Featured Researcher" area both flip-floppable and renameable?

**Comment by geekscruff** [comment]
@vantuyls we're looking to host hyku's for a number of folks and yes, I deffo think featured collections  rather than featured researcher would be good for some ... so maybe we're talking about a 'featured' feature where there are a number of different types of featured 'things' to choose from?

**Comment by hannahfrost** [comment]
At Stanford, we also like to feature deposited works themselves. So I think we have yet a third possible thing to promote via the widget.

**Comment by hannahfrost** [comment]
Adding to the list of requirements:

- [ ] Provide the ability to encode links and add images to the Featured * space. (Plain, basically formatted text is plain and basic.)

**Comment by ggeisler** [comment]
See https://github.com/samvera-labs/hyku/issues/457 for some homepage redesign ideas. That ticket doesn't specifically address the requirements presented at the top of this ticket, which are focused on implementation details of the featured researcher block, but I think we can address those requirements better after we get more consensus on all the potential elements we want to offer for the repository homepage. 

In other words, when we better understand all of what we want to offer for customizing the homepage, then we can figure out the best way to provide configuration for it all, including the featured researcher block.

**Comment by DiemBTran** [comment]
The attached screenshots demonstrate what the current Featured Researcher workflow looks like.
This is the what the content block looks like on the homepage. Admin can use plain text, hyperlinks, and photos.
![1003 homepage](https://user-images.githubusercontent.com/29311858/122453415-3308f580-cf5f-11eb-91d3-9529f3ece32c.jpeg)
This is what the dashboard edit form looks like. This workflow requires the admin to know HTML tags, which can be a blocker for some institutions.
![1003 dashboard](https://user-images.githubusercontent.com/29311858/122453513-4ddb6a00-cf5f-11eb-99b2-a5b1bb649512.jpeg)
